.yarnrc.yml

```diff
@@ -45,4 +45,12 @@ plugins:
     spec: "@yarnpkg/plugin-interactive-tools"
   - ./.hooks/plugin-warning-logger.js
 
+supportedArchitectures:
+  os:
+    - darwin
```

Comment:
Nit: Might be worth mentioning that the above line is to enable builds on Mac OS? I had to look it up.